web/components/mint.tsx

```diff
@@ -388,15 +387,16 @@ export const Mint = () => {
                   {pendingVaultRewards.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 6 })} DIKO
                 </p>
               </div>
-              <button 
-                type="button" 
-                className={classNames(pendingVaultRewards === 0 && 'hidden',
-                  'bg-indigo-600 hover:bg-indigo-700 inline-flex items-center px-3 py-2 ml-4 text-sm font-medium leading-4 text-white border border-transparent rounded-md shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500')}
-                onClick={() => claimPendingRewards()}
-                disabled={pendingVaultRewards === 0}
-              >
-                Claim rewards
-              </button>
+              { pendingVaultRewards > 0 &&
```

Comment:
Extra space before `{` here.
Also, we used the ternary `{condition ? () : null}` throughout the code base, maybe we should stick to it? I'm no expert, just aiming for consistency here.

Reply:
I guess we don't need an `else` clause, probably remove the `null`s in other places

Reply:
Agree with @philipdesmedt , this construct is better that ternaries with `null`, and to achieve consistency it's better to refactor the other places.

Reply:
Can you explain how/why it's better? https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
JS wonderfulness 😂 , that's a compelling reason to not use the `&&` then. I'll apply the changes.